packages/commonwealth/client/scripts/controllers/server/sessions.ts

```diff
@@ -157,69 +259,87 @@ class SessionsController {
   }
 
   // Public signer methods
-  public async signThread({ community, title, body, link, topic }) {
-    const { session, action, hash } = await this.sign('thread', {
+  public async signThread(
+    address: string,
+    { community, title, body, link, topic }
+  ) {
+    const { session, action, hash } = await this.sign(address, 'thread', {
       community: community || '',
       title,
-      body,
+      body: encodeURIComponent(body),
       link: link || '',
       topic: topic || '',
     });
     return { session, action, hash };
   }
 
-  public async signDeleteThread({ thread_id }) {
-    const { session, action, hash } = await this.sign('deleteThread', {
+  public async signDeleteThread(address: string, { thread_id }) {
```

Comment:
Let me push a change to add this to the client (since it is UX impacting) but since we are not verifying deletions on the server (goal for this PR being to get client side content signed) it isn't critical to include.